templates/button.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Default template for Fieldmanager_Button
+ *
+ * @package Fieldmanager
+ */
+?>
+<input
+	class="fm-button-element button-secondary <?php echo esc_attr( $this->field_class ); ?>"
```

Comment:
The button type should be an argument rather than hardcoded `button-secondary`. That can be the default, but making it an argument gives the implementor the choice.